test/specs/rules/require-capitalized-constructors-new.js

```diff
@@ -0,0 +1,54 @@
+var Checker = require('../../../lib/checker');
+var expect = require('chai').expect;
+
+describe('rules/require-capitalized-constructors-new', function() {
+    var checker;
+
+    function baseCases() {
```

Comment:
So it's not necessary to run these tests on both the `true` and `allExcept` configurations? I based it on [test/specs/rules/require-capitalized-constructors.js](https://github.com/jscs-dev/node-jscs/blob/master/test/specs/rules/require-capitalized-constructors.js)